test/fixtures/accept/github.json

```diff
@@ -8,6 +8,12 @@
       "method": "GET",
       "path": "/repos/:name/:repo/contents/:path*/package.json",
       "origin": "https://${GITHUB_TOKEN}@${GITHUB_API}"
+    },
+    {
```

Comment:
so why do you need to add a rule to whitelist a folder after all?

Reply:
As in what's the use case for whitelisting directories? I'm not sure 😂 but at least if someone _did_ then the directory can't be traversed 😬

Reply:
Someone might have all their dependency files in a "safe" folder or it might be part of a meta-project in a monorepo.

Reply:
we usually do not add rules which we do not use. that's why adding only specific files' rules. customers then can edit this file per what they need to.

Reply:
Just to clarify this is only a rule in a text fixture to prove that the exploit is not possible. It's not modifying the `accept.json` we provide as a template for others to extend

Reply:
oh, I missed the file name. NP then